source/OptimaJet.DWKit.StarterApplication/Controllers/UsersController.cs

```diff
@@ -25,10 +23,10 @@ public class UsersController : BaseController<User>
         }
 
         [HttpGet("current-user")]
-        public IActionResult GetCurrentUser() {
+        public async Task<IActionResult> GetCurrentUser() {
             var currentUser = CurrentUser;
 
-            return Ok(currentUser);
+            return await base.GetAsync(currentUser.Id);
```

Comment:
Change User.GetByAuth0Id to call DefaultGet()

```
        public async Task<User> GetByAuth0Id(string auth0Id)
            => await DefaultGet()
                .Where(e => e.ExternalId == auth0Id)
                .FirstOrDefaultAsync();
```
Then we can use currentUser here and be consistent with a User anytime it is being used (i.e. it always has the correct included members).

Reply:
I almost made that change earlier, but now it seems the right way to solve it.

Reply:
I got rid of DefaultGet, because we want to utilize the builtin jsonapi `Get` so we can have dynamically `Included` relationships. There is a way to manually crawl through the the query params' relationships and build that query ourselves, but I didn't think it appropriate for right here.